rotkehlchen/db/dbhandler.py

```diff
@@ -4,6 +4,7 @@
 import re
 import shutil
 import tempfile
+from collections import namedtuple
```

Comment:
Can you use `NamedTuple` from typing instead?

`from typing import NamedTuple`